kong/cmd/utils/prefix_handler.lua

```diff
@@ -378,6 +378,54 @@ local function write_process_secrets_file(path, data)
   return true
 end
 
+local function pre_create_lmdb(conf)
+  local prefix = conf.prefix
+  local lmdb_path = conf.lmdb_environment_path or "dbless.lmdb"
+  local user = string.match(conf.nginx_user or "", "%w+") or "$USER"
```

Comment:
Here I have a worry about the user name, can it works well in all conditions?

Reply:
The only alternative I see is doing a hard error here, if we can't extract a user from `conf.nginx_user`.